pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -747,17 +747,6 @@ func makeGvk(kind string) schema.GroupVersionKind {
 	}
 }
 
-func vapForVersion(gvk *schema.GroupVersion) (client.Object, error) {
```

Comment:
I agree with @maxsmythe on

> Actually, reflecting on moving vapForVersion and vapBindingForVersion, I think those should stay with the controller code.

Let's not move there two methods.

Reply:
done